config/routes.rb

```diff
@@ -35,6 +35,10 @@
   # Is this being used?
   get '/video', to: 'video#show'
 
+  namespace :user do
```

Comment:
We might want to think about a GitHub specific route & controller now that it is starting to take shape as an object?

Reply:
For sure. Let's see what these next stories ask.